cmd/aries-agent-mobile/pkg/wrappers/rest/endpoints.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package rest
+
+import (
+	"net/http"
+
+	"github.com/hyperledger/aries-framework-go/cmd/aries-agent-mobile/pkg/wrappers"
+)
+
+// Endpoint describes the fields for making calls to external agents
+type Endpoint struct {
+	Path   string
+	Method string
+	PIID   string
```

Comment:
variable stutter , PID should be enough ..

Reply:
hey Talwinder, I'm referring to the field representing a protocol instance id as these do: https://github.com/hyperledger/aries-framework-go/search?q=piid&unscoped_q=piid